public/app/plugins/panel/news/rss.ts

```diff
@@ -0,0 +1,28 @@
+import { RssFeed, RssItem } from './types';
+
+const CORS_PROXY = 'https://cors-anywhere.herokuapp.com/';
```

Comment:
Is this a trusted proxy? Feels a bit dangerous to use this one otherwise. If you add an feedUrl containing sensitive information this proxy will have access to all that information.

Reply:
Good point @mckn -- my plan was to remove this when our feed has the right CORS headers set and *then* move this panel from 'alpha' to general release.

But, I went ahead and changed the structure so we explicitly show the "herokuapp.com" in the default URL and will replace it when our server is setup for this.